### PR TITLE
Move add action on Trigger inside Trigger Detail Fragment #113

### PR DESCRIPTION
**Pull request by pg301**
Move add action on Trigger inside Trigger Detail Fragment #113

### CODE REVIEW

File: mobile/src/main/java/org/hawkular/client/android/Animae/MyBounceInterpolator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
```
Comment (danielpassos) [inline mobile/src/main/java/org/hawkular/client/android/Animae/MyBounceInterpolator.java]: 1. Never use camel case for package name
1. Let's rename the package for _animation_

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java [hunk 2]
```diff
@@ -16,33 +16,90 @@
...
+import butterknife.OnClick;
 import icepick.State;
 
 
 public class TriggerDetailFragment extends android.support.v4.app.Fragment {
             @State
             @Nullable
             Trigger trigger;
+    @BindView(R.id.add_button)
+    ImageButton button;
```
Comment (danielpassos) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java]: Let's avoid to use generic names like _button_. Let's use something that really represent button action like _addButton_

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java [hunk 3]
```diff
@@ -16,33 +16,90 @@
...
+
+    @BindView(R.id.text_title)
+    TextView textView;
```
Comment (danielpassos) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java]: Same for this one!

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java [hunk 4]
```diff
@@ -16,33 +16,90 @@
...
+    }
+
+    private void setUpBindings() {
+        ButterKnife.bind(this, getView());
     }
 
     @Override public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
+
+        setUpBindings();
```
Comment (danielpassos) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java]: I wanna start to kill this kind of unecessarly methods like I did here: https://github.com/hawkular/hawkular-android-client/pull/129

So... I'll like suggest you to kill this method and move the content to `onCreateView`

```java
public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
    View view =  inflater.inflate(R.layout.fragment_trigger_detail, container, false);
    ButterKnife.bind(this, view);
    return view;
}
```

File: mobile/src/main/res/drawable/ic_favorite_black_24dp.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<!--
```
Comment (danielpassos) [inline mobile/src/main/res/drawable/ic_favorite_black_24dp.xml]: Should remove `black_24dp` from the name of the file. Should be `ic_favorite`

File: mobile/src/main/res/layout/fragment_trigger_detail.xml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        android:layout_width="60dp"
+        android:layout_height="60dp"
+        android:layout_marginLeft="10dp"
+        android:scaleType="fitXY"
+        android:id="@+id/add_button"
+        android:background="@color/background_window"
+        android:src="@drawable/ic_favorite_black_24dp"
+         />
+
+</LinearLayout>
```
Comment (danielpassos) [inline mobile/src/main/res/layout/fragment_trigger_detail.xml]: Add newline at the end of the file

**Review by danielpassos — CHANGES_REQUESTED** [review]

**Comment by shubhamvashisht** [comment]
@pg301 waiting for this issue to be resolved so that i can start working on issue assigned to me.

**Comment by pg301** [comment]
@danielpassos @garg-anuj  I have done the requested changes, please have a look.

P.S. Sorry for the delay in response.

**Comment by pilhuhn** [comment]
@pg301 I think it would be better to give a quick summary of changes as you did in other comments and not only write "Changes in #113". This way it is easier in the commit log to understand what is done.

**Comment by pg301** [comment]
@pilhuhn Thanks for the advice sir, will keep this in mind.

**Comment by pg301** [comment]
@garg-anuj @danielpassos I think the issue is fixed. Please have a look and let me know if some more changes are required.

**Comment by KeenWarrior** [comment]
@pg301 Can you please also attach gif for this PR?

**Review by KeenWarrior — APPROVED** [review]
Looking good to me

**Comment by pg301** [comment]
demo1_new
@garg-anuj here's the gif

**Comment by KeenWarrior** [comment]
@danielpassos any comments?

**Review by danielpassos — APPROVED** [review]

**Comment by pg301** [comment]
@danielpassos @garg-anuj Thanks :)

**Pull request merged**